Frameworks/Foundation/NSFileManager.mm

```diff
@@ -379,7 +390,8 @@ - (BOOL)replaceItemAtURL:(NSURL*)originalItemURL
 }
 
 /**
- @Status Stub
+ @Status NotInPlan
+ @Notes Windows systems do not have a trash, we use the Recycle Bin
```

Comment:
Can this note just not be itself?

That's like saying "We don't throw things away in this house, we recycle them"

But the user still wants to know where the detritus goes.